python-sdk/tests/test_example_dags.py

```diff
@@ -51,38 +49,31 @@ def session():
     return get_session()
 
 
-DAG_BAG = DagBag(Path(__file__).parent.parent / "example_dags", include_examples=False)
-AIRFLOW_VERSION_INDICATOR = "airflow_version:"
-MINIMUM_AIRFLOW_VERSION = "2.2.5"
+def get_dag_folder() -> Path:
```

Comment:
hmm I don't like the Airflow versions in folder tbh :( 

I would rather prefer a list of dags that needs to be skipped for certain airflow versions in tests

Reply:
But that is not possible if we check for import errors, I believe. We cannot exclude files from the dag_bag dag folder.

Reply:
Or we dynamically create an `.airflowignore` :/

Reply:
but this would still mean we need to add the version information somehow/somewhere outside of the DAG.